code/modules/power/battery_ion.dm

```diff
@@ -0,0 +1,70 @@
+// Generic battery machine
+// stores power
+
+/obj/machinery/power/battery/portable
+	name = "portable power storage unit"
+	desc = "A IOn-model portable storage unit, used to transport charge around the station."
```

Comment:
No, largely because I'm not sure how easily that's going to be to balance.
<s>The name is also a pun on i/o and ion batteries, it probably doesn't use a chem gradient</s>
The content changes (setting charge levels, buildability and availability) will be in the next PR.